data/org.freedesktop.portal.Print.xml

```diff
@@ -24,20 +24,20 @@
 -->
 
 <node name="/" xmlns:doc="http://www.freedesktop.org/dbus/1.0/doc.dtd">
+  <interface name="org.freedesktop.portal.PrintRequest">
+    <signal name="Response">
+      <arg type="u" name="response"/>
+      <arg type="a{sv}" name="options"/>
+    </signal>
```

Comment:
I wonder if we should have a "standard" response like this in org.freedesktop.portal.Request.